ckanext/datitrentinoit/model/mapping.py

```diff
@@ -269,3 +269,26 @@ def _extras_as_dict(extras):
             extras_as_dict.append({'key': key, 'value': value})
 
     return extras_as_dict
+
+
+def create_opencity_package_dict(harvest_object, config):
+    json_dict = json.loads(harvest_object.content)
+
+    package_dict = {
+        "title": json_dict["title"],
+        "short_description": json_dict["short_description"],
+        "identifier": json_dict["identifier"],
+        "geographical_geonames_url": json_dict["spatial_coverage"],
+        "url": json_dict["uri"],
+    }
+
+    extras = {
+        "temporal_coverage": [
+            {
+                "temporal_start": json_dict["start_date"],
+                "temporal_end": json_dict["end_date"],
```

Comment:
Pls check the issue #79 about date encoding

Reply:
Done